SQL/updates/4-5.py

```diff
@@ -0,0 +1,103 @@
+# Listen up. 
```

Comment:
Make this 5-6, and the next 6-7 due to an SQL change being made since this PR, and the version needs to be bumped

Reply:
This file is **still** 4-5.py @Aurorablade